packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -3119,7 +3120,7 @@ export const audiusBackend = ({
     const waudioBalance = await audiusLibs.solanaWeb3Manager.getWAudioBalance(
       address
     )
-    if (waudioBalance == null) {
```

Comment:
TIL that `BigInt(0) == null` evaluates to `true`, big L. So I went and replaced all the == null and != null checks added in #6185 with `isNullOrUndefined` checks.

Reply:
Sad. Would we have been saved by `===`?